docs/spec.md

```diff
@@ -29,7 +29,7 @@ map :: Functor f => Type f ~> (a → b, f a) → f b
 ```
 
 We use syntax similar to Haskell's. You can learn about it from
-[Ramda's wiki](https://github.com/ramda/ramda/wiki/Type-Signatures) or from book
+[Ramda's wiki](https://github.com/ramda/ramda/wiki/Type-Signatures) or from the book
 ["Professor Frisby's Mostly Adequate Guide to Functional Programming"](https://drboolean.gitbooks.io/mostly-adequate-guide/content/ch7.html)
```

Comment:
+1
I tend to not add a `.` after a link sometimes because it looks ugly to me. But I guess that's just silly :)